### ISSUE TITLE
Bot with Viruses?

### ISSUE THREAD

**Issue by kic10** [ztizzlegaming/CSGOWinBig#221]
The bot is full with viruses and when i try to compile it my windows is auto-deleting them. Btw those are Trojans and other dangerours viruses, like wtf?

**Comment by andrewda** [comment]
No. It's not a virus. Windows's built-in virus detection is a bit too sensitive, so you can safely ignore those warnings.

**Comment by kic10** [comment]
I still can not compile it, it shows me problems with SteamAuth, can you help me maybe via TeamViewer?
